### DISCUSSION TITLE
Happy new year 🎉

### DISCUSSION THREAD

**Discussion by marcusolsson** [marcusolsson/obsidian-projects#300] [Announcements]
I want to thank you all for for the warm reception of the Projects plugin since its October release. Much has happened since, and most of it has been driven by your feedback. In this post, I'd like to look back at what's happened since the release, as well as look ahead for what's coming.

Please consider nominating Obsidian Projects for [Obsidian Gems of the Year](https://airtable.com/shrMd4Dj4oLgL55yt) (until Jan 15th) 🙏

## Looking back

Obsidian Projects took home the first place for the [Best new plugin](https://forum.obsidian.md/t/obsidian-october-2022-winners/49087) during Obsidian October! 🎉

To name a few of the features you've helped launch:

- Drag and drop support for all the views
- Filtering notes per views
- Conditional coloring for each note
- Predefined options for text fields
- Configurable image fitting in Gallery (thanks @Acylation!)
- Setting default project
- Listing notes that couldn't be parsed
- Sorting table columns by date (thanks @GamerGirlandCo!)
- Excluding notes from projects

... along with bug fixes and other enhancements!

However, one of the things I was most excited about didn't even happen in this repository. [Database folder](https://github.com/RafaelGB/obsidian-db-folder) was the first plugin to support the [Custom Views API](https://github.com/marcusolsson/obsidian-projects/tree/main/obsidian-projects-types). Thanks to @RafaelGB for doing the hard work in making this happen! 💪

## Looking forward

While new features certainly look great in the changelog, one of the things I'm most passionate about is delivering a great user experience. I'll continue to prioritize bug reports, but I'd also like to encourage you to create issues for more "annoying" parts of the plugin. Things that aren't necessarily broken, but still cause frustration. No issue is too small.

This plugin is my first project using Svelte, and while I'm happy about the decision, I've learned a lot about building an application in Svelte. I'll be looking at refactoring the application to remove glitches, focus issues, and other annoyances that bring the overall quality down.

In the short-term, you'll see some new features, but I'm planning to address some technical debt to improve the foundation of the plugin going forward.

In the long-term, I want to focus on improving the core platform. The more the platform can provide, the easier it's going to be to build and prototype new views. For example, improved data handling, schema detection, and error handling.

## Shout-outs

Thank you to all of you who've [bought me a coffee](https://buymeacoffee.com/marcusolsson). Your support keeps motivating me to continue maintaining the project!

Thank you to all of those who have created both videos and articles to showcase Projects! Be sure to check them all out in the [README](https://github.com/marcusolsson/obsidian-projects#learn-more).

Finally, I'd like to give a special thank you to a few contributors in particular that have been helping me and other users out by both triaging issues as well as submitting pull requests. Thank you for all your help @imeed166, @Acylation, @GamerGirlandCo, @GitMurf, and @Quorafind! 🙏 🙏

**Comment by RafaelGB** [comment]
Happy new year @marcusolsson !
Thanks for your hard work too and for makes Obsidian even better.

**Reply by marcusolsson** [reply]
Same! 💪
